pkg/logging/api.go

```diff
@@ -0,0 +1,25 @@
+// Package logging API for structured logging
+package logging
+
+// The global logger instance
+var logger Logger //nolint: gochecknoglobals
+
+// InitLogger initialize the global logger instance
+func InitLogger(logFile string) error {
```

Comment:
Would be nice to not have to initialize the logger. Requiring the logger to be initialized prevents us from logging things that happens before the initialization and I want to be able to log eeeeeverything. I see we're initializing the logger in `o.Initialize`. Some commands doesn't even do `o.Initialize`. Maybe if we move the `InitLogger` call to `cmd/okctl/main.go`

Reply:
I need some way to initialize it in order to be able to test it. But I can have a go at moving it to `main.go`!

Reply:
We could use the singleton pattern.

Reply:
@deifyed Moved the logger initialization to `main.go`

Reply:
nice!